code/modules/vore/eating/digest_act_vr.dm

```diff
@@ -19,23 +19,23 @@
 /////////////
 // Some indigestible stuff
 /////////////
-/obj/item/weapon/hand_tele/digest_act(...)
+/obj/item/weapon/hand_tele/digest_act(var/atom/movable/item_storage = null)
 	return FALSE
-/obj/item/weapon/card/id/gold/captain/spare/digest_act(...)
+/obj/item/weapon/card/id/gold/captain/spare/digest_act(var/atom/movable/item_storage = null)
```

Comment:
Wot, why do this? `(...)` means 'accept any named args, they're all fine', which is fine since we're just going to return false no matter what anyway.

Reply:
They were runtiming about item_storage being missing or something while also failing to return false to the thing trying to digest them. One of my tests happened to involve a cap gun, which would use these as a gun subtype, and instead of the gurgle attempt telling the belly "nope, can't gurgle." it would just rutime on each attempt without even adding it to the preserved items list to skip from next attempt.

Reply:
Anyhow yeah. This has been the reason behind the protected items misbehaving in bellies. These things runtiming about the (...) really does break the functionality. The items aren't surviving because this returning FALSE (which it fails to do due to the runtimes), but because this thing still overrides the part where the item would delete. They just get stuck looping in a failing proc that is overriding the deletion. No feedback is returned to the bellies to skip it or to add it to the ignore lists whatsoever. This is why the gurgles still keep going on and on and on without ever stopping if a protected item exists in a belly, and this is also why they never show up in the undigestable/uncleanable lists. For example, IDs and such in borg bellies end the cleanup with "Cleaning finished. # uncleanable objects pls eject." while the (...) items go with "Cleaning in progress. # (protected) items remaining..." being just stuck until the end of time.